app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -81,11 +89,257 @@
     background-size: 690px;
   }
 
+  // icon fallback
+  .no-svg & {
+    padding: $baseline-unit*2;
+  }
+
   .no-js &,
   .svg & span.icon.icon--web-chat,
   .no-svg & svg.icon-svg--web-chat {
     display: none;
   }
+
+  // popup opened state
+  & > *:not(.mobile-webchat--icon) {
+    opacity: 0;
+    transition: opacity 0.4s linear 0.1s;
+  }
+
+  &.mobile-webchat--opened {
+    padding: $baseline-unit*2;
+    width: 92vw;
```

Comment:
I think the width would be better expressed as a percentage so that it is more evenly positioned. In this case it could be something like `100% - ($default-gutter * 4)`.

Reply:
It does look more evenly positioned, thanks for the suggestion!